hapi-proto/src/main/proto/TransactionContents.proto

```diff
@@ -20,6 +22,12 @@
  * ‍
  */
 
-public enum SigStyle {
-	MAP, LIST
+option java_package = "com.hederahashgraph.api.proto.java";
+option java_multiple_files = true;
+
+import "BasicTypes.proto";
+
+message SignedTransaction {
+    bytes bodyBytes = 1; // TransactionBody serialized into bytes, which needs to be signed
+    SignatureMap sigMap = 2; // The signatures on the body with the new format, to authorize the transaction
```

Comment:
A silly question, seems like Transaction message will have only SignedTransaction as its in the future, why do we want introduce the SignedTransaction here to have another layer of indirection? I know if with Transaction alone, we have to keep in mind that it can also be a signed transaction, which I guess people already get used to.

Reply:
The answer for your question is in the description of the issue #432 itself

Reply:
I see.